docs/tenancy/1.x/affects-connections.md

```diff
@@ -99,3 +139,11 @@ class Role extends Model
     use OnTenant;
 }
 ```
+
+### Laravel DB Facade Example
+
+If you are using Laravel's `DB` Facade, you are able to access the tenant connection as seen in the example below.
+
+```php
+DB::connection('tenant')->select(...);
+```
```

Comment:
3 things:
- Let's add a small notice stating that this is not the recommended way.
- Let's change the example to use the facade to get the connection name `Tenancy::getTenantConnectionName()`
- Let's show that you can also get this connection from the Tenancy Facade / Environment `::getTenantConnection()`